web/src/compose_recipient.ts

```diff
@@ -302,13 +340,13 @@ function on_hidden_callback() {
     }
 }
 
-export function handle_middle_pane_transition() {
-    if (compose_state.composing) {
+export function handle_middle_pane_transition(): void {
+    if (compose_state.composing()) {
```

Comment:
Please put this apparent bug fix in a prep commit, since it's a behavior change.

Reply:
This has been fixed already. I have rebased to include the fix.